lms/djangoapps/completion/tests/test_models.py

```diff
@@ -142,3 +142,52 @@ def test_cannot_call_submit_completion(self):
                 completion=0.9,
             )
         self.assertEqual(models.BlockCompletion.objects.count(), 1)
+
+
+class SubmitBatchCompletionTestCase(TestCase):
+    """
+    Test that BlockCompletion.objects.submit_batch_completion has the desired
+    semantics.
+    """
+
+    def setUp(self):
+        super(SubmitBatchCompletionTestCase, self).setUp()
+        _overrider = waffle.waffle().override(waffle.ENABLE_COMPLETION_TRACKING, True)
+        _overrider.__enter__()
+        self.addCleanup(_overrider.__exit__, None, None, None)
+
+        blocks = [
+            (UsageKey.from_string('block-v1:edx+test+run+type@video+block@doggos'), 1.0),
+        ]
+        self.course_key = 'course-v1:edx+test+run'
+        self.course_key_obj = CourseKey.from_string(self.course_key)
```

Comment:
This can be condensed to:
```python
self.course_key = CourseKey.from_string('course-v1:edx+test+run')
...
CourseEnrollmentFactory.create(user=self.user, course_id=unicode(self.course_key))
```
And then reference `self.course_key` instead of `self.course_key_obj` in the test methods below.